configs/default/NEBD-NBD_INFINITYAIOV2PRO.config

```diff
@@ -0,0 +1,103 @@
+# Betaflight / STM32F745 (S745) 4.2.11 Nov  9 2021 / 20:29:04 (948ba6339) MSP API: 1.43
+
+board_name NBD_INFINITYAIOV2PRO
+manufacturer_id NEBD
+
+# resources
+resource BEEPER 1 D13
+resource MOTOR 1 C09
+resource MOTOR 2 C08
+resource MOTOR 3 C07
+resource MOTOR 4 C06
+resource LED_STRIP 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 7 E08
+resource SERIAL_TX 8 E01
+resource SERIAL_RX 1 B07
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 8 E00
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C00
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_SCK 4 E12
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MISO 4 E13
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 D06
+resource SPI_MOSI 4 E14
+resource ADC_BATT 1 C01
+resource ADC_CURR 1 C02
+resource FLASH_CS 1 B00
+resource OSD_CS 1 A15
+resource GYRO_EXTI 1 B01
+resource GYRO_EXTI 2 D00
+resource GYRO_CS 1 E11
+resource GYRO_CS 2 B12
+
+# timer
+timer C09 AF2
+# pin C09: TIM3 CH4 (AF2)
+timer C08 AF2
+# pin C08: TIM3 CH3 (AF2)
+timer C07 AF2
+# pin C07: TIM3 CH2 (AF2)
+timer C06 AF2
+# pin C06: TIM3 CH1 (AF2)
+timer A09 AF1
+# pin A09: TIM1 CH2 (AF1)
+timer D13 AF2
+# pin D13: TIM4 CH2 (AF2)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin C09 0
+# pin C09: DMA1 Stream 2 Channel 5
+dma pin C08 0
+# pin C08: DMA1 Stream 7 Channel 5
+dma pin C07 0
+# pin C07: DMA1 Stream 5 Channel 5
+dma pin C06 0
+# pin C06: DMA1 Stream 4 Channel 5
+dma pin A09 0
+# pin A09: DMA2 Stream 6 Channel 0
+dma pin D13 0
+# pin D13: DMA1 Stream 3 Channel 2
+
+# feature
+feature RX_SERIAL
+feature OSD
+
+# serial
+serial 0 64 115200 57600 0 115200
+
+# master
+set gyro_to_use = BOTH
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_burst = AUTO
+set dshot_bitbang = OFF
```

Comment:
This should be removed

Reply:
@Asizon should we remove from here to:
```
HENA-TALONF4V2.config:set dshot_burst = OFF
HGLR-HGLRCF405.config:set dshot_burst = OFF
HGLR-HGLRCF745.config:set dshot_burst = OFF
MTKS-MATEKF405SE.config:set dshot_burst = OFF
NERC-NEUTRONRCG4AIO.config:set dshot_burst = OFF
RCTI-BEEROTORF4.config:set dshot_burst = OFF
```

Reply:
Nop, these are f411 boards and need bitbanging off